docs/src/Avatar.doc.js

```diff
@@ -49,6 +49,11 @@ card(
         defaultValue: 'check-circle',
         description: 'This is the icon shown only when "verified=true"',
       },
+      {
+        name: '__dangerouslyUseDefaultIcon',
```

Comment:
@nachofassini can we remove it from the docs, we don't want this to be publicly visible since we'll remove the prop soon.